### DISCUSSION TITLE
타입스크립트를 유니티 애니메이션 이벤트에 연결할 수 없나요?

### DISCUSSION THREAD

**Discussion by LeeYongSu1** [naverz/zepeto-studio-kor#793] [5. Q&A]
제페토 월드 패키지 버전
1.3.0

현재 설치되어 있는 제페토 앱 버전
3.12.1

구현하려는 기능
유니티 애니메이션 클립에 이벤트 추가

발생 문제
'StandardKart' AnimationEvent 'SetDrifting' on animation 'Hop' has no receiver! Are you missing a component?'
라는 문구와함께 오류 발생

구성화면
![유니티화면](https://user-images.githubusercontent.com/80555875/169966054-b0dfd917-2608-4629-bae0-73e264af83bb.png)
![애니메이션 이벤트](https://user-images.githubusercontent.com/80555875/169966066-8e0e873f-5bfa-483e-8b00-0f153bb15c1d.png)
![스크립트](https://user-images.githubusercontent.com/80555875/169966073-b683d94b-0d10-4d93-8e5b-5828cd1e4132.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 질문 주신 내용은 현재 지원하지 않습니다.
지원하기 위해서는 연관 작업이 많으므로 시간이 소요될 것 같습니다.
양해 부탁 드립니다.
감사합니다.
